columnphysics/icepack_tracers.F90

```diff
@@ -1289,7 +1289,18 @@ subroutine icepack_compute_tracers (ntrcr,     trcr_depend,    &
                trcrn(it) = atrcrn(it) / aicen
             else
                trcrn(it) = c0
-               if (it == nt_Tsfc) trcrn(it) = Tf  ! surface temperature
+               if (it == nt_Tsfc) then
+! tcraig, these old options should be deprecated
+! exist for bit-for-bit backwards compatibility in testing
+                  if (tfrz_option == "mushy_old" .or. &
+                      tfrz_option == "linear_salt_old" .or. &
+                      tfrz_option == "constant_old" .or. &
+                      tfrz_option == "minus1p8_old") then
+                     trcrn(it) = Tocnfrz  ! surface temperature
```

Comment:
Okay, but can we fix this fairly quickly?  E.g. do the testing now to confirm that this indeed is what changes answers, then fix it and start using updated output for testing going forward?  Maybe there are a few steps along the way, but hopefully only a few.

Reply:
We can do that.  But my concern is that we'll start to diverge from the results on Consortium Main.  We are still bit-for-bit for all tests.  If we diverge, it's going to be hard to confirm that when we sync up the Consortium and E3SM branches, that everything is working properly and we got all the merged code correctly.  What we could also think about doing is modify Consortium Main so it gets the Tf modifications.  Or we can let them diverge.